### PR TITLE
Fixed a typo in README.md

### PR DESCRIPTION
**Pull request by Hanso707**
### Why:

Fixed a typo

### What's being changed:
In README.md: 
'Step 3: Suggest changes'  --> 'Activity: Suggest changes' --> '5.'
Changed **commment** to **comment** 

### Check off the following:

- [x] For workflow changes, I have verified the Actions workflows function as expected.

- [x] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by Victor-aseko — APPROVED** [review]
Right way

**Review by Victor-aseko — APPROVED** [review]
It's fine

File: README.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ Now that you have explored the different ways you can review a pull request it i
 1. Find the `index.html` changes.
 1. Hover your cursor next to the line numbers on the left side of the page.
 1. Click the blue plus icon.
-1. After the commment form appears, click the **Insert a suggestion** button.
+1. After the comment form appears, click the **Insert a suggestion** button.
 1. Edit the suggestion.
 1. Click **Add a single comment**.
```
Comment (kejunsun) [inline README.md:138]: sss

**Comment by kejunsun** [comment]
aaa

**Comment by Avinash0213** [comment]
Pull request

**Review by heiskr — APPROVED** [review]

**Comment by heiskr** [comment]
Fixed in another PR :)

**Pull request closed without merging**